fem/transfer.cpp

```diff
@@ -1860,6 +1860,21 @@ L2ProjectionGridTransfer::L2ProjectionH1Space::AllocR()
    return R_local;
 }
 
+L2ProjectionGridTransfer::L2ProjectionH1Space::~L2ProjectionH1Space()
+{
+
+   delete M_LH_local_op;
+
+#ifdef MFEM_USE_MPI
+   delete pfes_ho_scalar;
+   delete pfes_lor_scalar;
+#else
+   delete fes_ho_scalar;
+   delete fes_lor_scalar;
+#endif
```

Comment:
I think it's safe to just make `pfes_ho_scalar`, `pfes_lor_scalar`, `fes_ho_scalar`, and `fes_lor_scalar` into `unique_ptr`'s?

Reply:
That could work, but we have usage of manual deletes in destructors in this file as well. I'm open to whichever is preferred.

Reply:
+1 for `unique_ptr`

Reply:
Pushed up. Also... if we bump mfem cpp standard up to 14, we can start using ``std::make_unique`` --- how cool would that be!

Reply:
Agreed! I hope we will bump to C++17 in the next release.